### PR TITLE
Belts and some select belt slot items no longer require a jumpsuit to wear

### PR DESCRIPTION
**Pull request by SchrodingersWolf**
## What Does This PR Do
All belts and some belt slot items now could be worn if you do not have a jumpsuit, and they also do not get unequipped when a jumpsuit is unequipped.

Full list of items:

- All belts, including the Judo Belt and Belt Defibs
- All emergency tanks (Oxygen, Nitrogen, Plasma) and specialised belt tanks (Plasma and Nitrogen)
- Katanas and Toy Katanas (NOT energy katanas)
- Stunprods
- And of course, inflatable ducks (How could I forget?)

Also comments out redundant code in regards to unequipping items in `human_update_icons.dm`.

## Why It's Good For The Game
It always makes me unsure why you can't wear a toolbelt while you don't have a jumpsuit, considering that you can do that easily IRL, so this PR aims to change that behaviour.

This also has the added benefit of making it so vox and plasmamen don't lose their tanks that easily - Permaing vox prisoners is always a chore and involves them getting toxins damage because they are cuffed (And thus cannot hold on to their N2 tanks) while the Warden has to change their jumpsuits into prison uniforms, thus dropping their tanks on the floor.

Katanas, toy katanas and stunprods also could be equipped in belts and back at all times, which implies some sort of strap that you could use to tie around your waist as well.

As for the redundant code, `/mob/living/carbon/human/unEquip` in `human_inventory.dm` already handles unequipping items, so the code in `human_update_icons.dm` should not longer be needed. 

Inventory code should also not be in icon code (Not to mention I can only get it to work by removing the code in there), but because I'm not sure if it would break anything else I left it in just commented out.

## Testing
Loaded up local server, put on a bunch of belts and belt slot items while being naked, then put on a jumpsuit and took it off again to verify that my belt slot items didn't get unequipped.

Did the same with belt items that are supposed to be unequippable and verified that they cannot be equipped, and taking off my jumpsuit while they are equipped causes them to fall on the floor.

Also tested unequipping and reequipping items as well as storage containers to ensure removing redundant code didn't break anything.

## Changelog
:cl:
tweak: Belts and certain belt slot items are now able to be worn without jumpsuits
/:cl:

### CODE REVIEW

File: code/game/objects/items.dm
Comment (Henri215) [inline code/game/objects/items.dm:45]: autodoc this and probably use snake_case for the name.

**Comment by Coolrune206** [comment]
Belts I can understand, they sit on your waist.
But the rest? What's holding up the oxygen tank? It presumably clips on, so... ow.

**Comment by SchrodingersWolf** [comment]
> Belts I can understand, they sit on your waist.
> 
> But the rest? What's holding up the oxygen tank? It presumably clips on, so... ow.

I assumed that the tanks also have a strap on them so you can wear them around your waist, but if it is that much of an issue I could make it so you need to attach cables for you to wear it without jumpsuits.

File: code/game/objects/items.dm
Comment (SchrodingersWolf) [reply code/game/objects/items.dm]: Variable turned into a flag, so this should be solved as well.

Comment (SchrodingersWolf) [reply code/game/objects/items.dm]: Latest commit converted the ignoreunder variable to a flag now.

**Review by S34NW — APPROVED** [review]
Awaiting team approval

**Review by lewcc — APPROVED** [review]

**Pull request merged**
